extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -81,9 +81,9 @@
     "pageview": "https://scomcluster.cxense.com/Repo/rep.gif?ver=1&typ=pgv&sid=!siteId&ckp=_client_id_&loc=_source_url_&rnd=_random_&ref=_document_referrer_&ltm=_timestamp_&wsz=_screen_width_x_screen_height_&bln=_browser_language_&chs=_document_charset_&col=_screen_color_depth_&tzo=_timezone_&cp_cx_channel=amp"
   },
   "deepbi": {
-    "base": "https://!endpoint/v1/pixel/!datasetKey/p.gif?accessKey=!accessKey&gdpr=!gdpr&page_href=_page_href_&referrer_href=_document_referrer_&page_title=_title_&amp=true",
-    "pageview": "https://!endpoint/v1/pixel/!datasetKey/p.gif?accessKey=!accessKey&gdpr=!gdpr&page_href=_page_href_&referrer_href=_document_referrer_&page_title=_title_&amp=true&event_type=page-open",
-    "event": "https://!endpoint/v1/pixel/!datasetKey/p.gif?accessKey=!accessKey&gdpr=!gdpr&page_href=_page_href_&referrer_href=_document_referrer_&page_title=_title_&amp=true&event_type=event"
+    "base": "https://api.deep.bi/v1/pixel/!datasetKey/p.gif?accessKey=!accessKey&gdpr=!gdpr&page_href=_source_url_&referrer_href=_document_referrer_&page_title=_title_&amp=true",
+    "pageview": "https://api.deep.bi/v1/pixel/!datasetKey/p.gif?accessKey=!accessKey&gdpr=!gdpr&page_href=_source_url_&referrer_href=_document_referrer_&page_title=_title_&amp=true&event_type=page-open",
+    "event": "https://api.deep.bi/v1/pixel/!datasetKey/p.gif?accessKey=!accessKey&gdpr=!gdpr&page_href=_source_url_&referrer_href=_document_referrer_&page_title=_title_&amp=true&event_type=event"
```

Comment:
> !gdpr -> true

Since gdpr is defined in deepbi.js as true, you need to make that change here.

Reply:
Thank you!